sycl/test/extensions/prefetch.cpp

```diff
@@ -0,0 +1,38 @@
+// RUN: %clangxx -fsycl-device-only -S %s -o - | FileCheck %s
+
+#include <sycl/sycl.hpp>
+
+char data[] = {0, 1, 2, 3};
+
+// CHECK: [[PREFETCH_STR:@.*]] = private unnamed_addr addrspace(1) constant [19 x i8] c"sycl-prefetch-hint\00", section "llvm.metadata"
```

Comment:
Are these manually written? If so, do we have an utility script to generate them automatically? If yes, one might need to disable instrumentation/use `-O1` to get a more readable IR.